src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -134,6 +139,9 @@ protected override ILspServices ConstructLspServices()
 
         static void AddHandlers(IServiceCollection services)
         {
+            services.AddRegisteringHandler<ImplementationEndpoint>();
```

Comment:
These weren't added before which caused the Implementation test to fail. I added a test below to prevent a recurrence of this.